.github/workflows/ci.yml

```diff
@@ -397,7 +400,7 @@ jobs:
         timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
```

Comment:
We could also switch the JDK version used by `setup-java-build` to compile, build, and test the project for this job once with Java 17 and once with Java 177 when it comes down to the `Integration Tas tests` iteration.